src/JackTrip.cpp

```diff
@@ -615,8 +615,14 @@ void JackTrip::onStatTimer()
     }
     bool reset = (0 == pkt_stat.statCount);
     RingBuffer::IOStat recv_io_stat;
-    if (!mReceiveRingBuffer->getStats(&recv_io_stat, reset)) {
-        return;
+    if (getBufferStrategy() != 3) {
```

Comment:
This looks unnecessary

Reply:
Looking at how this is presented I realize now that it's not clear what I mean. The if and else blocks are the same.

Reply:
thanks -- historical -- removed that